### ISSUE TITLE
add new content for /contributors

### ISSUE THREAD

**Issue by zoepage** [webcompat/webcompat.com#1933]
follow up for https://github.com/webcompat/webcompat.com/pull/1922#discussion_r152741368

Content context: https://github.com/webcompat/webcompat.com/issues/1149

- [x] content for all 5 sections (roles) (convert images to text & images, check for max size of 600px)
  - [x] Report a bug
  - [x] Diagnose a Bug
  - [x] Site outreach
  - [x] Build tools (referring to contributing to webcompat.com, spec work / contribution)*
  - [x] Events (referring to speaking at events, organizing events -> e.g. bug sprints + link to karlcows article about sprints + how to report bugs.)*
- [x] make sections linkable
- [x] add informative titles to links
- [x] remove `Table of Content`
- [x] move "team section" from /about to "join us"
- [x] add Ioanas bio to team: https://github.com/webcompat/webcompat.com/issues/1576#issuecomment-351375058 (⚠️ please respect the alphabetic order)
- [x] add benefits of contributing to the projects and add to "join us"
- [x] add routes + templates to points marked with *
- [x] add sub navigation in footer

**Comment by karlcow** [comment]
I don't think we should add title to links. 
see https://github.com/webcompat/webcompat.com/pull/1922#discussion_r159589127

> The objective of this technique is to demonstrate how to use a title attribute on an anchor element to provide additional text describing a link. The title attribute is used to provide additional information to help clarify or further describe the purpose of a link. If the supplementary information provided through the title attribute is something the user should know before following the link, such as a warning, then it should be provided in the link text rather than in the title attribute. 

https://www.w3.org/TR/WCAG20-TECHS/H33.html

We currently have for example `<li><a href="/contributors/reproduce-bug">Reproduce a Bug</a></li>` which does exactly what it says on the tin.

**Comment by zoepage** [comment]
>> The title attribute is used to provide additional information to help clarify or further describe the purpose of a link.

> I don't think we should add title to links.

This is the content part that also needs to be taken care of. I think, this could be a great addition to give more context to the user. If not, its at least not hurting anyone. So win-win :)

**Comment by karlcow** [comment]
@zoepage 

What would you suggest as a title for `<li><a href="/contributors/reproduce-bug">Reproduce a Bug</a></li>` to understand the "win-win"?

**Comment by karlcow** [comment]
@notabene What do you think? Are the `title` attributes useful in this case? And if yes what would you recommend we do?

**Comment by notabene** [comment]
@karlcow Actually `title` attribute is on the down road these days, as it serves only partly its purpose (as you quoted from the spec). It's useful for people with a mouse on a desktop browser because they can hover over it, but it brings nothing to screen reader users, keyboard users, mobile browser users.

My practice is this:

1. provide a link that is understandable as-is
2. provide a `title` that gives useful but not necessary information, as a plus for people using a mouse, as a plus but not strictly needed to get the use of the link.

(up to par with the WCAG spec)

eg. on https://nota-bene.org/De-2017-a-2018

```html
<a
  href="https://github.com/ffoodd/a11y.css/tree/webextension"
  title="collaboration sur une extension de test de quelques critères d'accessibilité"
  >
  collaboration
</a>
```

So basically you're both agreeing on the same thing, I'd say:

* It's not needed
* It's a nice plus, so it's a win-win situation.

My advice: don't add titles for the sake of adding titles, but as a means to provide "comfortable" additional information.

**Comment by zoepage** [comment]
> My advice: don't add titles for the sake of adding titles, but as a means to provide "comfortable" additional information.

👍 

@karlcow 
> What would you suggest as a title for <li><a href="/contributors/reproduce-bug">Reproduce a Bug</a></li> to understand the "win-win"?

I can imagine, something like "If you are interested in starting with working on reported bugs, verifying bugs is a great start." could be a great addition.

**Comment by karlcow** [comment]
@zoepage thanks.

You said during the last webcompat meeting, you needed help for the content. 
What did you mean? When I read the first bug up there 
* "convert images to text & images". I don't understand because I'm not into the refactor project as much as you are. 
* "check for max size of 600px".

Or are you saying that the only source of our content is the images in 
https://github.com/webcompat/webcompat.com/issues/1149#issuecomment-284589695 👀 and we need to transcribe this into text.

**Comment by zoepage** [comment]
> we need to transcribe this into text.

Yes. First step would be to transcribe the images, so its accessible and others are able to contribute. 
Second step would be to iterate over the content and see what we can improve.

> "check for max size of 600px".
The images are quite big, which also makes them tricky to display in responsive mode.
<img width="651" alt="screen shot 2018-01-15 at 12 18 20 pm" src="https://user-images.githubusercontent.com/562354/34940041-6025c4a2-f9ee-11e7-90a6-53d7ef3a2520.png">

With the new content navigation, we even have less space. This is why, if we use images in the content, we'd only use a max width of 600px. This way we won't loose too much detail as well as save bandwidth (e.g. for mobile).

**Comment by karlcow** [comment]
@zoepage we can't make images until the design is finished. This seems to be the last part. Maybe I need more chocolates or something, my brain is not processing what is being asked. :)

**Comment by karlcow** [comment]
For content, you would like new content so if I understood and not the ones from the mockups. Under which form do you want this content? Text only, HTML? in a PR in the right template if it already exists?
In this last case? Which html templates should I edit?

**Comment by zoepage** [comment]
@karlcow 
As the missing mockups seem to be an issue, let's break it down to "just" content.

1. Create a google doc
2. describe the roles / steps / work for each of the 5 roles.

Roles are (as described in https://github.com/webcompat/webcompat.com/issues/1933#issue-276433031):
- Report a bug
- Diagnose a Bug
- Site outreach
- Build tools (referring to contributing to webcompat.com, spec work / contribution)*
- Events (referring to speaking at events, organizing events -> e.g. bug sprints + link to karlcows article about sprints + how to report bugs.)*

Then we can review the content and move it to HTML. Happy to help with that.

**Comment by karlcow** [comment]
@zoepage 

hmm interesting. I have the feeling there are missing things and scopes not completely right.
The content has said in #1149 will end up in different pages. So there will be probably a `/contribute/` and subpages.

1. Report a Bug
2. Triage a Bug (not in the list) but it is part of https://github.com/webcompat/webcompat.com/issues/1149#issuecomment-284589695
3. Diagnose a bug
4. Site Outreach
5. Build Tools (contributing to webcompat.com. ok! but I don't think spec work fits in here)
6. Web platform research (specifications, reduced tests cases, etc.)
7. Organizing Webcompat events

Let's prioritize on things which are in the two lists of these last two comments.

**Comment by zoepage** [comment]
@karlcow that's exactly the track! :+1:
Let me know, if you want feedback :)

**Comment by karlcow** [comment]
* [x] [Report a Bug](https://gist.github.com/karlcow/d5254456df9bc722edbb0512e635fc83#file-report-bug-md)
* [x] [Triage a Bug](https://gist.github.com/karlcow/d5254456df9bc722edbb0512e635fc83#file-triage-bug-md) (not in the list) but it is part of https://github.com/webcompat/webcompat.com/issues/1149#issuecomment-284589695
* [x] [Diagnose a bug](https://gist.github.com/karlcow/d5254456df9bc722edbb0512e635fc83#file-diagnose-bug-md)
* [x] [Site Outreach](https://gist.github.com/karlcow/d5254456df9bc722edbb0512e635fc83#file-outreach-bug-md)
* [ ] Build Tools (contributing to webcompat.com. ok! but I don't think spec work fits in here)
* [ ] Web platform research (specifications, reduced tests cases, etc.)
* [ ] Organizing Webcompat events

**Comment by karlcow** [comment]
Yoohoo @zoepage first pass on the content is done.

**Comment by zoepage** [comment]
Note, so I don't forget...
We should move the "bios" from about to join us on contributors.

**Comment by zoepage** [comment]
@karlcow where shall I put the comments?

**Comment by karlcow** [comment]
@zoepage Do you mean comments on the content? I guess once it is in place. It will be easier to review inside a PR.

**Comment by zoepage** [comment]
Yes on the gist... You want to move it to a PR first or get comments other way?

**Comment by karlcow** [comment]
> It will be easier to review inside a PR.

Yes inside a PR than we can comment.
@zoepage Do you mean you want me to put them in the documents? Which templates/URIs does this live in the new design?

**Comment by zoepage** [comment]
That would be great @karlcow!

The content would go into following templates: 
https://github.com/webcompat/webcompat.com/tree/refactor/webcompat/templates/contributors

We'd need 3 more templates for 
- Build Tools (contributing to webcompat.com. ok! but I don't think spec work fits in here)
- Web platform research (specifications, reduced tests cases, etc.)
- Organizing Webcompat events

(Is this helpful or would you like to have more info?)

**Comment by karlcow** [comment]
@zoepage I addressed the request in #2056
Hope it works for you.

**Comment by zoepage** [comment]
we've merged this.

**Comment by zoepage** [comment]
As we're on the content review right now, I reopened the issue.

Collecting feedback:

`contributors/report-bug`
https://www.evernote.com/shard/s310/sh/afcb3634-58c6-4b9a-8a01-2574e942287f/dd82e2b592b5b0e8

**Comment by adamopenweb** [comment]
`staging.webcompat.com`
https://www.evernote.com/l/ATbi2V1l-t1BGrQdYt5b1x-B3JgNG9wstxo

**Comment by zoepage** [comment]
merged #2282
